CONTRIBUTORS.md

```diff
@@ -81,7 +81,7 @@ This list is manually curated to include valuable contributions by volunteers th
 | @ameeker | |
 | @StaggerLeee | |
 | @jblz | |
-| | @nicbertino |
+| | @nic-bertino |
```

Comment:
2nd column is WP.org profile. I guess it was okey: https://profiles.wordpress.org/nicbertino.

Reply:
Raaah, hah, will fix, thanks!